coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -94,7 +92,7 @@ export default class Certify extends Vue {
 </script>
 
 <style lang="scss" scoped>
-@import '@/assets/styles/theme.scss';
+// @import '@/assets/styles/theme.scss';
```

Comment:
Why is the theme commented out here?

Reply:
It's not needed in this section atm, but I left it (but commented out) for easier re-inclusion if we need to import any theme variables later.

Reply:
Ah ok, sounds good to me. I assume this is the case for all the commented out themes. ( Hence why i didn't comment on the rest of them )

Reply:
Yup, all unneeded ones are commented out atm.